doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,72 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformTargetRegressor` transforms the target before fitting a
+regression model and inverting back the prediction to the original space. It
+takes as an argument the regressor that will be used for prediction, and the
+transformer that will be applied to the target variable::
+
+  >>> import numpy as np
+  >>> from sklearn.datasets import load_boston
+  >>> from sklearn import preprocessing
+  >>> from sklearn.linear_model import LinearRegression
+  >>> from sklearn.model_selection import train_test_split
+  >>> boston = load_boston()
+  >>> X = boston.data
+  >>> y = boston.target
+  >>> transformer = preprocessing.StandardScaler()
+  >>> regressor = LinearRegression()
+  >>> regr = preprocessing.TransformTargetRegressor(regressor=regressor,
+  ...                                               transformer=transformer)
+  >>> X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+  >>> regr.fit(X_train, y_train) # doctest: +ELLIPSIS
+  TransformTargetRegressor(...)
+  >>> print('R2 score:', regr.score(X_test, y_test)) # doctest : +ELLIPSIS
+  R2 score: 0.63...
+
+The transformer can also be replaced by a function and an inverse function. We
```

Comment:
Rephrase:

"For simple transformations, instead of a `Transformer` object, a pair of functions can be passed, defining the transformation and its inverse mapping::"

because it's not clear to me what it means "the transformer can be replaced". Plus, it can't always: not when it's stateful.